app/services/youtube_service.rb

```diff
@@ -1,26 +1,31 @@
 class YoutubeService
-  attr_reader :keywords
+  attr_reader :tags, :keywords
 
-  def initialize(tags)
-    @tags = tags
+  def initialize(params)
+    @tags = params[:tags]
+    @title = params[:title]
+    @description = params[:description]
```

Comment:
I like this use of params much better. Going to allow for a more robust API call and allows for more dynamic search-ability in the case of updates to tags.